### PR TITLE
fix: syntax typo in sql article slug

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: redirects.json [hunk 1]
```diff
@@ -4,6 +4,11 @@
   { "from": "/archive/(.*)$", "to": "/", "permanent": false },
   { "from": "/our-sponsors", "to": "/sponsors", "permanent": true },
   { "from": "/about-freecodecamp", "to": "/about", "permanent": true },
+  {
+    "from": "/sql-order-by-statement-example-sytax",
+    "to": "/sql-order-by-statement-example-syntax",
```
Comment (ShaunSHamilton) [inline redirects.json:9]: Hey Kris,
At what point does this go through? That is, the `to` address is still a 404.

Comment (scissorsneedfoodtoo) [reply redirects.json:9]: Sorry for the late response @ShaunSHamilton, just seeing this now.

Once this is merged, I'll go ahead and change the slug on the live article from `sql-order-by-statement-example-sytax` to `/sql-order-by-statement-example-syntax`. Then I can deploy the new redirects.

This should minimize any downtime.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Review by naomi-lgbt — APPROVED** [review]

**Comment by nhcarrigan** [comment]
@scissorsneedfoodtoo will leave this for you to merge whenever you are ready to deploy :)

**Comment by scissorsneedfoodtoo** [comment]
Awesome, thanks for your reviews @ShaunSHamilton and @nhcarrigan.

Will merge and deploy this now.

**Pull request merged**